metricbeat/tests/system/test_base.py

```diff
@@ -121,7 +121,7 @@ def kibana_dir(self):
 
     def is_saved_object_api_available(self):
         kibana_semver = semver.VersionInfo.parse(self.get_version())
-        return kibana_semver.major == 7 and kibana_semver.minor < 15 or kibana_semver.major >= 8
+        return semver.compare(kibana_semver, "7.14.0") >= 0
```

Comment:
:+1: but maybe we don't need this check if we are also updating the environments.

Reply:
I would rather keep it to make sure we are not causing failures. I have checked our codebase, I haven't found references to older versions where there should not have been. But you know, better to be safe than sorry when it comes to the CI. :D